org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
In the description it says `The Drupal variable is referenced from the global window object. If Drupal is not defined, it won't break modules referencing this file.` but the code used is `Drupal.settings.currentCiviCRMUserId`. So if Drupal is not defined, we would get error right?